package.json

```diff
@@ -66,7 +66,6 @@
     "@openshift/dynamic-plugin-sdk-webpack": "1.0.0-alpha3",
```

Comment:
FWIW this is the package that should be updated ASAP (see todo here: https://github.com/openshift/hac-dev/pull/408/files#diff-4be71c3182a0de045ef56aec0e1aa8d5bf9075135637452164bda6e0cb015aa0R17)

I attempted to do so but ran into an issue where version `^2.0.0` of `@openshift/dynamic-plugin-sdk-webpack` requires `webpack` `^5.75.0` as a peer dependency,  however `webpack` comes from `@redhat-cloud-services/frontend-components-config` in hac-dev. This means  `@redhat-cloud-services/frontend-components-config` must be updated to the latest version. However in doing so, the production webpack build never completes and hangs indefinitely traversing the `node_modules` dir. I suspect it may be an issue with the sass loader.